### ISSUE TITLE
Run filters on folder button doesn't run filters

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/quickFilters#1]
"I have found that recently in ver 4.1, the run filters on folder Icon/ button has stop working.  Menu - Tools, Run filters on folder works."

**Comment by RealRaven2000** [comment]
This regression is caused by some code I wrote for Thunderbird 68 - code:
```
  if (util.versionGreaterOrEqual(util.AppverFull, "67")) {
    var {MailServices} = ChromeUtils.import("resource:///modules/MailServices.jsm"); // new module spelling
  }
  else 
    Components.utils.import("resource:///modules/mailServices.js", {});
```
the last line will import the Symbol as "mailServices" and not "MailServices"

Fix is to force the new variable name in both cases:
```
  var {MailServices} = 
    (util.versionGreaterOrEqual(util.AppverFull, "64")) ?
      ChromeUtils.import("resource:///modules/MailServices.jsm") : // new module spelling
      Components.utils.import("resource:///modules/mailServices.js", {});
```

**Comment by RealRaven2000** [comment]
Fixed in version [4.4.1](https://quickfilters.quickfolders.org/version.html#4.4.1).
